### PR TITLE
Adding Golden Tags, metrics and summaries for several entities

### PR DESCRIPTION
**Pull request by mfuentes-newrelic**
### Relevant information

Describe what you have done and any details that you think are relevant or that you might want to discuss with us. 

### Checklist

* [ ] I've read the guidelines and understand the acceptance criteria.
* [ ] The value of the attribute marked as `identifier` will be unique and valid. 
* [ ] I've confirmed that my entity type wasn't already defined. If it is I'm providing an
 explanation above.

### CODE REVIEW

**Review by cianBuckley — APPROVED** [review]

File: definitions/infra-awsecscluster/summary_metrics.yml [hunk 1]
```diff
@@ -1,13 +1,3 @@
-providerAccountName:
```
Comment (naxhh) [inline definitions/infra-awsecscluster/summary_metrics.yml:1]: Why are we removing this one? Most of the AWS summaries have them

**Comment by naxhh** [comment]
Hey @mfuentes-newrelic minor comments. I've also noticed there are conflicts in this PR and you need to sign the CLA if we want to merge this one

File: definitions/infra-awsecscluster/summary_metrics.yml (hunk 1 above)
Comment (mfuentes-newrelic) [reply definitions/infra-awsecscluster/summary_metrics.yml:1]: Good catch! Fixed!

File: definitions/infra-gcprouternatgateway/summary_metrics.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+newConnections:
+  title: New connections
+  UNIT: COUNT
```
Comment (naxhh) [inline definitions/infra-gcprouternatgateway/summary_metrics.yml:3]:
```suggestion
  unit: COUNT
```

I think there are several files with this. Remember you can run the linter to verify these things
https://github.com/newrelic-experimental/entity-synthesis-definitions/tree/main/validator#local-setup

File: definitions/infra-gcpkubernetescontainer/summary_metrics.yml [hunk 3]
```diff
@@ -1,19 +1,17 @@
...
+  unit: SECONDS
   query:
-    eventId: entityGuid
     select: average(`container.cpu.usageTime`)
     from: GcpKubernetesContainerSample
-  unit: SECONDS
-  title: Cumulative CPU utilization over 60s
-memoryUsage:
-  query:
+    facet: entityName
```
Comment (naxhh) [inline definitions/infra-gcpkubernetescontainer/summary_metrics.yml:7]: summary metrics don't support facets

File: definitions/infra-gcprouternatgateway/summary_metrics.yml (hunk 2 above)
Comment (mfuentes-newrelic) [reply definitions/infra-gcprouternatgateway/summary_metrics.yml]: Fixed!

File: definitions/infra-gcpkubernetescontainer/summary_metrics.yml (hunk 3 above)
Comment (mfuentes-newrelic) [reply definitions/infra-gcpkubernetescontainer/summary_metrics.yml]: Fixed!

**Comment by naxhh** [comment]
Since there has not been movement here I'll be closing this PR. 

If this is still relevant please feel free to rebase it and reopen.

**Pull request closed without merging**